frontend/packages/knative-plugin/console-extensions.json

```
@@ -1124,5 +1124,369 @@
+        "version": "v1",
+        "kind": "Configuration"
+      },
+      "color": "#005f60",
+      "label": "%knative-plugin~Configuration%",
+      "labelPlural": "%knative-plugin~Configurations%",
+      "abbr": "CFG"
+    }
+  },
+  {
+    "type": "console.model-metadata",
+    "properties": {
+      "model": {
+        "group": "operator.knative.dev",
+        "version": "v1alpha1",
+        "kind": "KnativeServing"
+      },
+      "color": "#005f60",
+      "label": "%knative-plugin~KnativeServing%",
+      "labelPlural": "%knative-plugin~KnativeServings%",
+      "abbr": "KS"
+    }
+  },
+  {
+    "type": "console.model-metadata",
+    "properties": {
+      "model": {
+        "group": "operator.knative.dev",
+        "version": "v1alpha1",
+        "kind": "KnativeEventing"
+      },
+      "color": "#005f60",
+      "label": "%knative-plugin~KnativeEventing%",
+      "labelPlural": "%knative-plugin~KnativeEventings%",
+      "abbr": "KE"
+    }
+  },
+  {
+    "type": "console.model-metadata",
+    "properties": {
+      "model": {
+        "group": "serving.knative.dev",
+        "version": "v1",
+        "kind": "Revision"
+      },
+      "color": "#005f60",
+      "label": "%knative-plugin~Revision%",
+      "labelPlural": "%knative-plugin~Revisions%",
+      "abbr": "REV"
+    }
+  },
+  {
+    "type": "console.model-metadata",
+    "properties": {
+      "model": {
+        "group": "serving.knative.dev",
+        "version": "v1",
+        "kind": "Route"
+      },
+      "color": "#005f60",
+      "label": "%knative-plugin~Route%",
+      "labelPlural": "%knative-plugin~Routes%",
+      "abbr": "RT"
+    }
+  },
+  {
+    "type": "console.model-metadata",
+    "properties": {
+      "model": {
+        "group": "serving.knative.dev",
+        "version": "v1",
+        "kind": "Service"
+      },
+      "color": "#005f60",
+      "label": "%knative-plugin~Service%",
+      "labelPlural": "%knative-plugin~Services%",
+      "abbr": "KSVC"
+    }
+  },
+  {
+    "type": "console.model-metadata",
+    "properties": {
+      "model": {
+        "group": "serving.knative.dev",
+        "version": "v1alpha1",
+        "kind": "DomainMapping"
+      },
+      "color": "#005f60",
+      "label": "%knative-plugin~DomainMapping%",
+      "labelPlural": "%knative-plugin~DomainMappings%",
+      "abbr": "DM"
+    }
+  },
+  {
+    "type": "console.model-metadata",
+    "properties": {
+      "model": {
+        "group": "sources.eventing.knative.dev",
+        "version": "v1alpha1",
+        "kind": "CronJobSource"
+      },
+      "color": "#005f60",
+      "label": "%knative-plugin~CronJobSource%",
+      "labelPlural": "%knative-plugin~CronJobSources%",
+      "abbr": "CJS"
+    }
+  },
+  {
+    "type": "console.model-metadata",
+    "properties": {
+      "model": {
+        "group": "sources.knative.dev",
+        "version": "v1beta2",
+        "kind": "PingSource"
+      },
+      "color": "#005f60",
```

```suggestion
      "color": "#7d1007",
```
color is different for eventing and serving resources, this color applies to all eventing resources